pkg/kapp/logs/pod_container_log.go

```diff
@@ -76,8 +100,7 @@ func (l PodContainerLog) Tail(ui ui.UI, cancelCh chan struct{}) error {
 		line, err := reader.ReadBytes('\n')
 		if err != nil {
 			if err == io.EOF {
-				ui.BeginLinef("%s# ending tailing '%s' logs\n", linePrefix, l.tag)
-				return nil
+				return err
```

Comment:
I am curious as to why we didn't check for `l.opts.Follow` (i.e. container restarting) here and return nil after sleeping for 500ms.

Reply:
Adopted